backend/census_historical_migration/workbooklib/end_to_end_core.py

```diff
@@ -191,19 +191,19 @@ def api_check(json_test_tables):
     return combined_summary
 
 
-def generate_workbooks(user, email, dbkey, year):
+def generate_workbooks(user, dbkey, year):
     entity_id = "DBKEY {dbkey} {year} {date:%Y_%m_%d_%H_%M_%S}".format(
         dbkey=dbkey, year=year, date=datetime.now()
     )
     sac = setup_sac(user, entity_id, dbkey)
     if sac.general_information["audit_type"] == "alternative-compliance-engagement":
         print(f"Skipping ACE audit: {dbkey}")
```

Comment:
Definitely could become a log statement, and perhaps should become a logged "failure to migrate" when we see ACEE audits (for now).